public/css/module.less

```diff
@@ -72,3 +81,9 @@ img.icon {
   overflow: hidden;
   display: inline-block;
 }
+
+.issue-table {
```

Comment:
Is that wrapping needed?

Reply:
I'm just thinking why `span` and then adding `display: block` :)

Reply:
Now you mention it, I think it makes sense to just use `div`. And remove the styling.

Reply:
`p` would be more fitting, if you ask me :sweat_smile:

Reply:
Changed it to `p`